hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/operations/CustomExtCodeSizeOperation.java

```diff
@@ -53,7 +58,7 @@ public OperationResult execute(@NonNull final MessageFrame frame, @NonNull final
                 frame.pushStackItem(UInt256.ZERO);
                 return new OperationResult(cost(true), null);
             }
-            if (proxyUpdaterFor(frame).contractMustBePresent() && !addressChecks.isPresent(address, frame)) {
+            if (contractRequired(frame, address, featureFlags) && !addressChecks.isPresent(address, frame)) {
```

Comment:
by this point I'm beginning to wish that `contractRequired(...) && !addressChecks.isPresent(...)` had been given a method with an informative name ....

Reply:
true I could name it something like `possiblyInvalidSolidityAddress`.  I'll make this change if there are other comments that need fixing.